odh/overlays/moc/monitoring/dashboards/operate-first/jupyterhub-usage.yaml

```diff
@@ -0,0 +1,919 @@
+              "value": "aakankshaduggal745-40gmail-2ecom"
+            },
+            "datasource": "opendatahub",
+            "definition": "kubelet_volume_stats_used_bytes{namespace=\"opf-jupyterhub\"}",
+            "hide": 0,
+            "includeAll": false,
+            "label": "User ID",
+            "multi": false,
+            "name": "user_id",
+            "options": [
+              {
+                "selected": true,
+                "text": "aakankshaduggal745-40gmail-2ecom",
+                "value": "aakankshaduggal745-40gmail-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "aasthana-40redhat-2ecom",
+                "value": "aasthana-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "aduggal-40redhat-2ecom",
+                "value": "aduggal-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "eran-2eraichstein-40gmail-2ecom",
+                "value": "eran-2eraichstein-40gmail-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "fmurdaca-40redhat-2ecom",
+                "value": "fmurdaca-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "hnalla-40redhat-2ecom",
+                "value": "hnalla-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "hveeradh-40redhat-2ecom",
+                "value": "hveeradh-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "kachauha-40redhat-2ecom",
+                "value": "kachauha-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "mcliffor-40redhat-2ecom",
+                "value": "mcliffor-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "mhild-40redhat-2ecom",
+                "value": "mhild-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "mpovolny-40redhat-2ecom",
+                "value": "mpovolny-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "ochatter-40redhat-2ecom",
+                "value": "ochatter-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "sbadhe-40redhat-2ecom",
+                "value": "sbadhe-40redhat-2ecom"
+              },
+              {
+                "selected": false,
+                "text": "shanand-40redhat-2ecom",
+                "value": "shanand-40redhat-2ecom"
+              }
+            ],
+            "query": "kubelet_volume_stats_used_bytes{namespace=\"opf-jupyterhub\"}",
+            "refresh": 0,
```

Comment:
I think you can just set `refresh: 1` and it will generate the `User ID` list above.